src/libmongoc/src/mongoc/mongoc-client-private.h

```diff
@@ -240,6 +240,10 @@ mongoc_client_connect (bool buffered,
                        const mongoc_uri_t *uri,
                        const mongoc_host_list_t *host,
                        bson_error_t *error);
+
+MONGOC_EXPORT (bool)
+mongoc_client_uses_server_api (mongoc_client_t *client);
```

Comment:
Should probably have prefix '_'.

Reply:
All of these functions are private since they are in a private header. There is already inconsistency for use of `_` prefix on private functions. Either way seems OK to me.